### DISCUSSION TITLE
post/image-processing-server-framework/

### DISCUSSION THREAD

**Discussion by giscus** [kakaopay/techlog-comments#30] [Announcements]
# post/image-processing-server-framework/

얼굴 인식, OCR 등 실생활에서의 문제를 해결하기 위해서는 이미지 처리를 위한 서버가 필요합니다. 이미지 처리를 담당하는 서버의 프레임워크를 선정하기 위한 테스트 내용과 결과를 공유합니다.

https://tech.kakaopay.com/post/image-processing-server-framework/

**Comment by underdarks** [comment]
파이썬 프레임워크들중 어떤것을 사용해야할지에 대해서  고민하던중에 좋은글을 발견하게 되어서 도움이 될것같습니다!  감사합니다!!

**Comment by junnn327** [comment]
파이썬 웹 프레임워크들의 성능 차이에 대해 궁금했는데 명쾌하게 정리되어 있어서 아주 유익했습니다! 감사합니다!

**Comment by jinwooseok** [comment]
우왓..! fast api의 특징이 궁금해서 들어왔는데 비동기 프레임워크였군요.. drf를 사용하고 있는데 fast api가 빠르다는 말이 실제 테스트 결과가 있으니까 더욱 확신이 드네요.. 잘 읽었습니다..공부해야겠다.. 감사합니다~!
